docs/operator-manual/declarative-setup.md

```diff
@@ -676,8 +676,10 @@ extended to allow assumption of multiple roles, either as an explicit array of r
   }
 ```
 
-Example service account configs for `argocd-application-controller` and `argocd-server`. Note that once the annotations
-have been set on the service accounts, both the application controller and server pods need to be restarted.
+Example service account configs for `argocd-application-controller` and `argocd-server`.
+
+!!! warning
+    Once the annotations have been set on the service accounts, both the application controller and server pods need to be restarted.
```

Comment:
I think for it to render as a warning box, it should rather be like this

```suggestion
> [!WARNING]
>  Once the annotations have been set on the service accounts, both the application controller and server pods need to be restarted.
```

Reply:
I used a few other warning messages in Argo CD doc as reference. Actually I cannot find `[!WARNING]` being used

<img width="627" alt="image" src="https://github.com/argoproj/argo-cd/assets/14064342/0cda1c85-8b43-4f7f-bc19-d6b968568f63">
<img width="435" alt="image" src="https://github.com/argoproj/argo-cd/assets/14064342/be98b72c-9fbd-448d-86aa-374434f2869a">

Reply:
Yeah you're right, was looking at Github Markdown syntax rather than readthedocs.